LOAN/Loans/LoansParties.rdf

```diff
@@ -69,79 +93,34 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-loan-loan-pty;borrows.1"/>
-				<owl:onClass rdf:resource="&fibo-loan-ln-ln;LoanContract"/>
+				<owl:onClass rdf:resource="&fibo-loan-ln-ln;Loan"/>
 				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-loan-pty;mayBe"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-gty;LoanGuarantor"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-tem-app;hasAssessmentInformation"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-tem-app;LoanApplicationBorrowerAssessmentInformation"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-tem-apr;mayBecome"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-pty;LoanBorrower"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">cosigner</rdfs:label>
 		<skos:definition xml:lang="en">Someone who commits to fulfil the obligations of the Borrower in the event that the Borrower defaults on the loan.</skos:definition>
 		<skos:editorialNote xml:lang="en">A co-borrower is different than a cosigner in that a cosigner takes responsibility for the debt should the borrower default, but does not have ownership in the property. Co-borrowers are frequently spouses or partners who use their combined income to qualify for a larger mortgage than could be obtained singularly, and are willing to share the risk of default on the mortgage and the general risks of homeownership. Review session notes 19 May 2011 usually identified by some identifier such as SSN, whether individual or a corporation. Contact information is at legal entity level. co-borrower or co-signer. Are these synonymous? Sometimes. Also have a concept of &quot;Joint Borrower&quot; e.g. a married couple. The legal entity which is the Co-borrower may become the Borrower under certain circumstances. Student Loans: The student may be the borrower, or the parent may be the borrower on behalf of the student. Student would not normally be a co-borrower but in theory it could. The co-borrower is someone with more established credit. Other factors: legal age - you need ot be of legal age in order to sign the Loan Contract, this is because you have to be a Legal Entity. so then the adult is simply the Borrower. Also potentially have references on the loan. These would come in handy later. These may be related to the student or the borrower.</skos:editorialNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-loan-loan-pty;JointBorrower">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-loan-pty;LoanBorrower"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-dbt;Borrower"/>
```

Comment:
If there are multiple Borrowers on a Loan are they not by definition JointBorrowers? What is the point of this class? Is one designated as primary? How?

Reply:
I'll ask Bruce to review when we get there.  Banks often designate a primary account holder and possibly primary borrower if there are multiples, but I don't know that we need this class.